models/hospitalAdministrator.js

```diff
@@ -8,8 +10,14 @@ module.exports = (sequelize, DataTypes) => {
      * The `models/index` file will call this method automatically.
      */
     static associate(models) {
-      // define association here
-      // TODO
+      HospitalAdministrator.belongsTo(Hospital, {
+        foreignKey: {
+          name: "hospitalId",
```

Comment:
So, part of the reason I've been making all of these naming convention suggestions is so that we _may_ not need to have all of these foreignKey attributes specified if we've got our models and attributes named appropriately- I think it _may_ just work to say ```HospitalAdministrator.belongsTo(models.Hospital);```

If not (I'll merge in the test framework and create a sample unit test to demonstrate), then we would add the foreignKey name... but we don't need to add more than that because Sequelize will pick up the column attributes on the linked model definition.

Reply:
That would be nice. I'll stay tuned!